bsp_diff/caas/device/intel/civ/host/vm-manager/0006-Revert-Upgrade-the-qemu4.2-to-6.0-and-add-SRIOV-chan.patch

```diff
@@ -0,0 +1,132 @@
+index 57d55ed..667a37c 100755
+--- a/scripts/setup_host.sh
++++ b/scripts/setup_host.sh
+@@ -9,7 +9,7 @@ set -eE
+ 
+ #---------      Global variable     -------------------
+ reboot_required=0
+-QEMU_REL="qemu-6.0.0"
++QEMU_REL="qemu-4.2.0"
+ CIV_WORK_DIR=$(pwd)
+ CIV_GOP_DIR=$CIV_WORK_DIR/GOP_PKG
+ CIV_VERTICAl_DIR=$CIV_WORK_DIR/vertical_patches/host
+@@ -39,16 +39,15 @@ function ubu_install_qemu_gvt(){
+     sudo apt purge -y "^qemu"
+     sudo apt autoremove -y
+     sudo apt install -y git libfdt-dev libpixman-1-dev libssl-dev vim socat libsdl2-dev libspice-server-dev autoconf libtool xtightvncviewer tightvncserver x11vnc uuid-runtime uuid uml-utilities bridge-utils python-dev liblzma-dev libc6-dev libegl1-mesa-dev libepoxy-dev libdrm-dev libgbm-dev libaio-dev libusb-1.0-0-dev libgtk-3-dev bison libcap-dev libattr1-dev flex libvirglrenderer-dev build-essential gettext libegl-mesa0 libegl-dev libglvnd-dev libgl1-mesa-dev libgl1-mesa-dev libgles2-mesa-dev libegl1 gcc g++ pkg-config libpulse-dev libgl1-mesa-dri
+-    sudo apt install -y ninja-build libcap-ng-dev
+ 
+     [ ! -f $CIV_WORK_DIR/$QEMU_REL.tar.xz ] && wget https://download.qemu.org/$QEMU_REL.tar.xz -P $CIV_WORK_DIR
+     [ -d $CIV_WORK_DIR/$QEMU_REL ] && rm -rf $CIV_WORK_DIR/$QEMU_REL
+     tar -xf $CIV_WORK_DIR/$QEMU_REL.tar.xz
+ 
+     cd $CIV_WORK_DIR/$QEMU_REL/
++    patch -p1 < $CIV_WORK_DIR/patches/qemu/Disable-EDID-auto-generation-in-QEMU.patch
+     patch -p1 < $CIV_WORK_DIR/patches/qemu/0001-Revert-Revert-vfio-pci-quirks.c-Disable-stolen-memor.patch
+     patch -p1 < $CIV_WORK_DIR/patches/qemu/0002-qemu-change-fence-poll-time-by-current-workload.patch
+-    patch -p1 < $CIV_WORK_DIR/patches/qemu/0003-Disable-EDID-auto-generation-in-QEMU.patch
+     if [ -d $CIV_GOP_DIR ]; then
+         for i in $CIV_GOP_DIR/qemu/*.patch; do patch -p1 < $i; done
+     fi
+@@ -93,7 +92,7 @@ function ubu_build_ovmf_gvt(){
+     sudo apt install -y uuid-dev nasm acpidump iasl
+     cd $CIV_WORK_DIR/$QEMU_REL/roms/edk2
+ 
+-    patch -p1 < $CIV_WORK_DIR/patches/ovmf/0001-OvmfPkg-add-IgdAssignmentDxe.patch
++    patch -p4 < $CIV_WORK_DIR/patches/ovmf/OvmfPkg-add-IgdAssgingmentDxe-for-qemu-4_2_0.patch
+     if [ -d $CIV_GOP_DIR ]; then
+         for i in $CIV_GOP_DIR/ovmf/*.patch; do patch -p1 < $i; done
+         cp $CIV_GOP_DIR/ovmf/Vbt.bin OvmfPkg/Vbt/Vbt.bin
+@@ -117,7 +116,7 @@ function ubu_build_ovmf_gvt(){
+ 
+     source ./edksetup.sh
+     make -C BaseTools/
+-    build -b DEBUG -t GCC5 -a X64 -p OvmfPkg/OvmfPkgX64.dsc -D NETWORK_IP4_ENABLE -D NETWORK_ENABLE  -D SECURE_BOOT_ENABLE -D TPM_ENABLE
++    build -b DEBUG -t GCC5 -a X64 -p OvmfPkg/OvmfPkgX64.dsc -D NETWORK_IP4_ENABLE -D NETWORK_ENABLE  -D SECURE_BOOT_ENABLE -DTPM2_ENABLE=TRUE
+     cp Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd ../../../OVMF.fd
+ 
+     if [ -d $CIV_GOP_DIR ]; then
+@@ -184,28 +183,6 @@ function ubu_enable_host_gvt(){
+     fi
+ }
+ 
+-function ubu_enable_host_sriov(){
```

Comment:
The sriov change is introduced by patch 0001-Support-SRIOV-in-setup-and-start-scripts.patch, which should only be merged into ADL branch,. Why does TPY branch pick up this branch?